frontend/src/screens/learner/card.tsx

```diff
@@ -15,17 +15,17 @@ interface StudyCardProps {
 }
 
 const Card = styled(Box)({
-    minWidth: 400,
-    maxWidth: 400,
+    minWidth: 272,
```

Comment:
you should double check with @dkhowala about this, this will affect the card size throughout the entire application and we may not want this to be global, but if we want to change the general card size anyway this should be fine

Reply:
I will check with devika.